sieve/bytecode.h

```diff
@@ -715,7 +715,8 @@ enum bytecode_tags {
 enum bytecode_imip_bitflags {
     IMIP_UPDATESONLY    = 1<<0,
     IMIP_DELETECANCELED = 1<<1,
-    IMIP_INVITESONLY    = 1<<2
+    IMIP_INVITESONLY    = 1<<2,
+    IMIP_ALLOWPUBLIC    = 1<<3
```

Comment:
If you keep a trailing comma on the last one, then next time you add one you don't need to edit the one above too.

```suggestion
    IMIP_INVITESONLY    = 1<<2,
    IMIP_ALLOWPUBLIC    = 1<<3,
```

Reply:
Yep. I go back and forth on that. Some people complain about a trailing comma on the last element of an array. I like to do it for the reason you stated - cleaner diff

Reply:
It's a syntax issue in some language I don't really use much... javascript, maybe, or python?  But we're not using that language :)